sections/03_categorize.md

```diff
@@ -540,7 +540,7 @@ adversarial neural networks as a tool to make sharable EHR data
 model from such attacks.
 
 There are approaches to protect models, but they pose their own challenges.
-Training algorithms in a differentially private manner provides a limited
+Training algorithms in a differentially private manner provide a limited
```

Comment:
I still think "training".  Remove "algorithms" from the sentence and it still works:

> Training in a differentially private manner provides a limited...